src/Altinn.Platform/Altinn.Platform.Events/Events/Controllers/EventsController.cs

```diff
@@ -15,30 +19,30 @@ namespace Altinn.Platform.Events.Controllers
     [ApiController]
     public class EventsController : ControllerBase
     {
-        private readonly IEventsRepository _repository;
+        private readonly IEventsService _postgresService;
```

Comment:
maybe rename to _eventsService? controller should not referece postgresql

Reply:
Total agree!